README.md

```diff
@@ -32,3 +52,5 @@ Run the ./refine file
 ```
 ./refine
 ```
+
+You can find more information on the [FAIRifier wiki](https://github.com/DTL-FAIRData/FAIRifier/wiki).
```

Comment:
no its an old version of the readme and also some other pages. Should i add information to the wiki main page?